index.js

```diff
@@ -50,6 +63,16 @@
 // // => "different"
 // console.log("%c----------", "color: red") 
 
+function sameSameButDifferent(num, maybeNum) {
+  if(num === maybeNum){
+  return "same same";
+} elsif (num == maybeNum){
+      return "same same (but different)";
+    } else{
+    return "different"
+  }
+}
+
```

Comment:
This is great. Please keep an eye on your indentation so your code stays organized.
You can try playing around with parseInt to avoid using the ==

To go deeper you can use typeof to check the datatype of the variables as well